_layouts/home.html

```diff
@@ -13,12 +13,99 @@ <h2 id="countdown">&nbsp;</h2>
     </p>
   </div>
 
-  <div class="flex row-xs-center ">
-    <div class="btn-group-lg">
-      <a class="btn btn-lg btn-default btn-next btn-xs-full" href="{{ '/host/' | prepend: site.baseurl }}">Become a host&nbsp;&nbsp;<i class="fa fa-play" aria-hidden="true"></i></a>
-      <a class="btn btn-lg btn-default btn-next btn-xs-full" href="{{ '/participant/' | prepend: site.baseurl }}">Attend an event&nbsp;&nbsp;<i class="fa fa-play" aria-hidden="true"></i></a>
-    </div>
-  </div>
+  <!-- Add this css File in head tag-->
+          <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" media="all">
+          <link href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css" rel="stylesheet" media="all">
+
+          <div id="bootstrap-touch-slider" class="carousel bs-slider fade  control-round indicators-line" data-ride="carousel" data-pause="hover" data-interval="5000" >
+
+              <!-- Indicators -->
+              <ol class="carousel-indicators">
+                  <li data-target="#bootstrap-touch-slider" data-slide-to="0" class="active"></li>
+                  <li data-target="#bootstrap-touch-slider" data-slide-to="1"></li>
+                  <li data-target="#bootstrap-touch-slider" data-slide-to="2"></li>
+              </ol>
+
+              <!-- Wrapper For Slides -->
+              <div class="carousel-inner" role="listbox">
+
+                  <!-- Third Slide -->
+                  <div class="item active">
+
+                      <!-- Slide Background -->
+                      <img src="../img/homepage_slide#1.jpg" alt="Bootstrap Touch Slider"  class="slide-image"/>
+                      <div class="bs-slider-overlay"></div>
+
+                      <div class="container">
+                          <div class="row">
+                              <!-- Slide Text Layer -->
+                              <div class="slide-text slide_style_left">
+                                <h1 data-animation="animated zoomInRight">Looking forward to meet you in our conference!!!</h1>
+                                <p data-animation="animated fadeInLeft">The best is yet to come!!!</p>
+                                <a href="http://bootstrapthemes.co/" target="_blank" class="btn btn-default" data-animation="animated fadeInUp">Become an host</a>
```

Comment:
Oh, I see, the links to "Become a host" and "Attend an event" would be here. However, this would break the link to the subsites of our website. Check out the code on the left side of the diff here to see where those were pointing to before your change. We definitely don't want to link to bootstrapthemes.co though ;)